tests/unittests/UIKit/UIFontTests.mm

```diff
@@ -108,8 +108,8 @@
 
 TEST(UIFont, Metrics) {
     float size = 22.0f;
-    UIFont* font = [UIFont fontWithName:@"Segoe UI Semibold Italic" size:size];
-    ASSERT_OBJCEQ(@"Segoe UI Semibold Italic", [font fontName]);
+    UIFont* font = [UIFont fontWithName:@"Segoe UI" size:size];
+    ASSERT_OBJCEQ(@"Segoe UI", [font fontName]);
 
     ASSERT_NE(0, [font descender]);
```

Comment:
out of scope for this review - this change is both time-sensitive, and due to the nature of the bug, I'd like to minimize risk and keep this change as small as possible. we can address other things if tests pass in the lab tonight, so please file non-showstopper feedback as separate issues.

---
In reply to: [85243923](https://github.com/Microsoft/WinObjC/pull/1234#discussion_r85243923) [](ancestors = 85243923)